network-api/networkapi/buyersguide/factory.py

```diff
@@ -190,6 +214,22 @@ def generate(seed):
         worst_case='Duplicate work that burns through screenshots'
     )
 
+    print('Generating fixed Buyer\'s Guide SoftwareProduct for visual regression testing')
+    SoftwareProductFactory.create(
+        blurb='Visual Regression Testing',
+        company='Percy',
+        draft=False,
+        email='percy@example.com',
+        live_chat='https://example.com/percy/chat',
+        name='percy cypress app',
+        phone_number='1-555-555-5555',
+        price=350,
```

Comment:
```suggestion
        price='| Free',
```

if we wanna make this fake product slightly realistic with our [SoftwareProduct use cases](https://foundation.mozilla.org/en/privacynotincluded/categories/video-call-apps/).